src/components/views/Chart.js

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from 'react'
```

Comment:
@jsbean this is the component that uses Victory to draw the charts if you're interested in looking into it. My sense is we should switch over to just drawing a custom chart on an html canvas or at least using vanilla d3 for the long term. And we'll need a whole bunch of new designs to accommodate all the possible question sets anyhow.